### PR TITLE
Minor edit for Reach Out on docs home page

### PR DESCRIPTION
**Pull request by stevenGravy**
Grammar edit for Reach out to and added line to try out hosted teleport

### CODE REVIEW

File: docs/pages/index.mdx [hunk 1]
```diff
@@ -50,7 +50,10 @@ Teleport is a Certificate Authority and an Access Plane for your infrastructure.
     Create an issue in GitHub.
   </TileListItem>
   <TileListItem href="https://goteleport.com/get-started/">
-    Reach out sales for a demo of Teleport Enterprise.
+    Reach out to sales for a demo of Teleport Enterprise.
+  </TileListItem>
+  <TileListItem href="https://goteleport.com/get-started/">
```
Comment (zmb3) [inline docs/pages/index.mdx:55]: Should we really be linking to the same place for both tiles?

Comment (stevenGravy) [reply docs/pages/index.mdx]: changed it to the signup link

File: docs/pages/index.mdx [hunk 2]
```diff
@@ -50,7 +50,10 @@ Teleport is a Certificate Authority and an Access Plane for your infrastructure.
     Create an issue in GitHub.
   </TileListItem>
   <TileListItem href="https://goteleport.com/get-started/">
-    Reach out sales for a demo of Teleport Enterprise.
+    Reach out to sales for a demo of Teleport Enterprise.
+  </TileListItem>
+  <TileListItem href="https://goteleport.com/signup/">
+    Try Teleport hosted for you.
```
Comment (zmb3) [inline docs/pages/index.mdx:56]: "hosted for you" is a bit confusing for me. Not sure if it means self-hosted or cloud.

I suspect you mean cloud, in which case you should just say "Try Teleport Cloud"

Comment (ptgott) [inline docs/pages/index.mdx:56]:
```suggestion
Start a free trial of Teleport Cloud.
```

**Review by ptgott — APPROVED** [review]
Left a suggestion and agree with Zac on the wording of the "Try Teleport hosted for you" line.   Approved so you're not blocked

File: docs/pages/index.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/index.mdx]: used @ptgott 's suggestion

**Comment by stevenGravy** [comment]
@zmb3 we should be good here.

**Review by r0mant — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**